man/subscription-manager.8

```diff
@@ -604,7 +604,8 @@ option
 .B --remove=NAME
 Removes a named override from the repositories specified with the
 .B --repo
-option
+option.
+Source of the override that is attempted to be removed may be one of the environment(s) the system belongs in. It can only be removed by either: removing the override from the environment it originates from or removing the system from the environment the override originates from.
```

Comment:
The first sentence could be a bit more straightforward, like: `The override will not be removed if its source is one of the environments the system belongs in. It can only...`